config/config.go

```diff
@@ -107,5 +108,12 @@ func Load(filename string) (*Config, error) {
 		conf.ExportURL = DefaultExportURL
 	}
 
+	if conf.TmpDir == "" {
+		conf.TmpDir, err = ioutil.TempDir("", "hauser")
```

Comment:
Is this a safe change? What was happening for empty string folks before? Storing in the relative current directory?

Reply:
Ah, it looks like it does just resolve to the current directory (makes sense 😄 ). I can remove this change.